apps/nextjs-website/src/components/atoms/ChatInputText/ChatInputText.tsx

```diff
@@ -0,0 +1,59 @@
+import { SendOutlined } from '@mui/icons-material';
+import { IconButton, InputBase, Paper, useTheme } from '@mui/material';
+import { useTranslations } from 'next-intl';
+import { ChangeEvent, FormEvent, useState } from 'react';
+
+type ChatInputTextProps = {
+  onSubmit: (message: string) => null;
+};
+
+const ChatInputText = ({ onSubmit }: ChatInputTextProps) => {
+  const t = useTranslations();
+  const [message, setMessage] = useState('');
+  const { palette } = useTheme();
+  const iconColor = palette.grey[700];
```

Comment:
This const is used only once i think is better to asing value directly in this case